### PR TITLE
Artwork button updates and removes anchor wrapping ArtworkListItem

### PR DESCRIPTION
**Pull request by kierangillen**
- The `ArtworkListItem` component had been using a wrapper outside of the component for linking to the artist. I've moved this `<TouchableWithoutFeedback/>` inside of the component and deleted it as an external wrapper in all cases.

- Updates buttons to use the new Palette `<Button/>` in `PartnerCard` and `ArtworkListItem`

- Updates render in `AboutArtist` to not render text section unless text is available.

Screen Shot 2019-06-26 at 3 19 48 PM

#trivial

### CODE REVIEW

File: src/lib/Components/ArtistListItem.tsx [hunk 1]
```diff
@@ -159,36 +152,30 @@ export class ArtistListItem extends React.Component<Props, State> {
   render() {
     const { isFollowedChanging } = this.state
     const { artist } = this.props
-    const { image, name } = artist
-    const { is_followed } = artist
+    const { is_followed, initials, image, href } = artist
+    const { url } = image
 
     return (
-      <Flex justifyContent="space-between" alignItems="center" flexDirection="row">
```
Comment (kierangillen) [inline src/lib/Components/ArtistListItem.tsx]: All of this code can be deleted and simply replaced by `<Avatar/>`

File: src/lib/Components/ArtistListItem.tsx [hunk 2]
```diff
@@ -159,36 +152,30 @@ export class ArtistListItem extends React.Component<Props, State> {
...
-            grayBorder={true}
-            text={is_followed ? "Following" : "Follow"}
-            onPress={this.handleFollowArtist}
-            selected={is_followed}
-            inProgress={isFollowedChanging}
-          />
-        </Box>
-      </Flex>
+      <Theme>
+        <TouchableWithoutFeedback onPress={() => this.handleTap(href)}>
```
Comment (kierangillen) [inline src/lib/Components/ArtistListItem.tsx]: Adds `<TouchableWithoutFeedback/>` to have anchor inside of component.

File: src/lib/Scenes/Artwork/Components/AboutArtist.tsx [hunk 3]
```diff
@@ -11,23 +11,31 @@ interface AboutArtistProps {
 
 export class AboutArtist extends React.Component<AboutArtistProps> {
   render() {
-    const hasSingleArtist = this.props.artwork.artists.length === 1
+    const {
+      artwork: { artists },
+    } = this.props
+    const hasSingleArtist = artists && artists.length === 1
+    const text =
```
Comment (kierangillen) [inline src/lib/Scenes/Artwork/Components/AboutArtist.tsx]: checks for text before rendering text section.

**Comment by sweir27** [comment]
This _looks_ fine to me, but I don't have a ton of context on the buttons (and especially the small changes you made to analytics/tapping behavior) so we should make sure to verify!

**Pull request merged**